clients/lsp-ada.el

```diff
@@ -57,25 +60,191 @@ environment."
 ;;;###autoload(put 'lsp-ada-project-file 'safe-local-variable 'stringp)
 
 (lsp-defcustom lsp-ada-option-charset "UTF-8"
-  "The charset to use by the Ada Language server. Defaults to \\='UTF-8\\='."
+  "The charset to use by the Ada Language server.  Defaults to \\='UTF-8\\='."
   :type 'string
   :group 'lsp-ada
   :package-version '(lsp-mode . "6.2")
   :lsp-path "ada.defaultCharset")
+;;;###autoload(put 'lsp-ada-option-charset 'safe-local-variable 'stringp)
```

Comment:
Do we need to autoload all these custom variables? 🤔

Reply:
The reason I added them is that any of those could potentially be specified in a .dir-locals.el.  I find it annoying to have to manually approve the "potentially unsafe" settings in my .dir-locals.el.  If you think this is too much, I can cut these back to the most frequently used options.  I could of course add safety settings in my init.el, but since other people use this mode as well, it would require others to do the same in their init.el, and I was going for easy of use.

Reply:
Let's cut back for now since it's unusual for only this file to do that. 🤔 We can open a new issue, discuss this, and see what others think.